templates/repo/home.tmpl

```diff
@@ -53,7 +53,11 @@
 		{{end}}
 		{{if .Repository.IsArchived}}
 			<div class="ui warning message">
-				{{.locale.Tr "repo.archive.title"}}
+				{{if .Repository.ArchivedUnix.IsZero}}
+					{{.locale.Tr "repo.archive.title_no_date"}}
+				{{else}}
+					{{.locale.Tr "repo.archive.title_date" (DateTime "long" .Repository.ArchivedUnix.FormatLong .Repository.ArchivedUnix.FormatShort) | Safe}}
```

Comment:
I found a conflict ... after the `DateTime` fix:

* #24317

These `DateTime` calls should be updated .....

Reply:
That's not a conflict (yet)

Reply:
The linked PR needs to be merged first before I can update it

Reply:
Merge this first then?

Reply:
Now it's a conflict

Reply:
I have now updated this PR